### PR TITLE
data: JMD110

### PR DESCRIPTION
**Pull request by JMD110**

### CODE REVIEW

**Comment by SaraJaoude** [comment]
Please change the file name so that it matches your GitHub username exactly. 

Instead of: `data/dellaji.json` this should be `data/JMD110.json`

You may wish to refer to point 7 of our QuickStart Guide in Docs: https://linkfree.io/docs/quickstart

File: data/dellaji.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      ],
+    "links": [
+        {
+            "name": "Follow me on Github",
+            "url": "https://github.com/JMD110",
+            "icon": "FaGithub"
+        },
+        {
+            "name": "Follow me on Linkedin",
+            "url": "https://www.linkedin.com/in/della-ji-ba6519110/",
```
Comment (jaydeepkhatri) [inline data/dellaji.json:22]: Can you please verify the Linkedin URL?
I got a 404 when I tried to visit the page.

Thanks :)

**Review by jaydeepkhatri — CHANGES_REQUESTED** [review]

**Comment by JMD110** [comment]
> Please change the file name so that it matches your GitHub username exactly.
> 
> Instead of: `data/dellaji.json` this should be `data/JMD110.json`
> 
> You may wish to refer to point 7 of our QuickStart Guide in Docs: https://linkfree.io/docs/quickstart

Thanks. The file name have been replaced.

File: data/dellaji.json (hunk 1 above)
Comment (JMD110) [reply data/dellaji.json]: Sorry for the confusion. Please try again. [linkin](https://www.linkedin.com/in/della-ji-ba6519110)

Comment (SaraJaoude) [reply data/dellaji.json]: @JMD110 I don't think that your LinkedIn link is working still, but as this is not a blocker I will merge your Pull Request. 

If you want to make any changes or additions to your Profile, then please raise another PR.  I would suggest you might want to take a look at the Editing Guide in Docs when you do: https://linkfree.io/docs/how-to-guides/editing

**Pull request merged**